core/app/models/spree/return_authorization.rb

```diff
@@ -61,6 +62,10 @@ def can_cancel_return_items?
 
     private
 
+      def authorize_return
+        order.authorize_return! unless order.nil?
```

Comment:
When did you decide to go forward with Interactor? Just curious on your motivation.